packages/schema/src/types/nuxt.ts

```diff
@@ -52,6 +52,7 @@ export interface NuxtPlugin {
   ssr?: boolean
   src: string
   mode?: 'all' | 'server' | 'client'
+  order?: number
```

Comment:
Maybe we could have some jsdoc on this to note our predefined numbers in [plugin-metadata.ts](https://github.com/nuxt/nuxt/pull/21611/files#diff-cd49fde9efc76ee9562216ef2ae79223c8c134c9fdb1de5c68a03922d0d22a86)?